index/postings_test.go

```diff
@@ -354,6 +354,26 @@ func TestRemovedPostings(t *testing.T) {
 
 }
 
+func TestRemovedNextStackoverflow(t *testing.T) {
+	var full []uint64
+	var remove []uint64
+
+	var i uint64
+	for i = 0; i < 1e7; i++ {
+		full = append(full, i)
+		remove = append(remove, i)
+	}
+
+	flp := newListPostings(full)
+	rlp := newListPostings(remove)
+	rp := newRemovedPostings(flp, rlp)
+	gotelem := false
+	for rp.Next() {
+		gotelem = true
+	}
```

Comment:
`testutil.Ok(t, rp.Err())` missing.

`gotelem` -> `gotElem` for readability.

Reply:
Done.